TcpKeepAliveSettings-reproducer/TcpKeepAliveSettings/TcpKeepAliveSettings.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="AMQPNetLite.Core" Version="2.3.0" />
```

Comment:
I am not sure if this is right. I think that this line will have effect of downloading and using upstream version of library. You should probably change reference to absolute/relative path of our DLLs from clients zip.